### PR TITLE
Image creation

### PR DESCRIPTION
**Pull request by bvanwijk**
Preset images are now created at one place in the code.
When a preset is removed from the system, the image will also be removed.

### CODE REVIEW

File: backend/src/main/java/com/benine/backend/preset/Preset.java [hunk 1]
```diff
@@ -157,6 +170,47 @@ public boolean equals(Object o) {
...
+   * @throws IOException  If the image cannot be written.
+   * @throws SQLException if the image can not be saved in the database.
+   */
+  public void createImage(StreamReader streamReader, String folder) throws
+          StreamNotAvailableException, IOException, SQLException {
+
+    File path = getNewImagePath(folder);
+
+    VideoFrame snapShot = streamReader.getSnapShot();
+    MJPEGFrameResizer resizer = new MJPEGFrameResizer(160, 90);
```
Comment (jochem725) [inline backend/src/main/java/com/benine/backend/preset/Preset.java]: Group goto-fail asked if we could adjust the snapshot size to be larger, maybe we can change this to `(640,360)`.

**Comment by jochem725** [comment]
Looks much better, I think we can merge this.

File: backend/src/main/java/com/benine/backend/preset/Preset.java (hunk 1 above)
Comment (bvanwijk) [inline backend/src/main/java/com/benine/backend/preset/Preset.java]: @jochem725 I've changed the image size to 640 x 360, but I have also added the image size to the config so it's now easy to adjust.

**Comment by jochem725** [comment]
@bvanwijk Great, I like the config option.

**Comment by Kingdorian** [comment]
Looks really good. But it might be good to add a comment to the config file in the lines above the height and width specifing what the range can be. I get an exception when I enter a negative or a big number.

**Comment by NaomidR** [comment]
Looks good! Cleaner than it was before!

**Comment by bvanwijk** [comment]
@Kingdorian I have added comments about the range in the config file

**Comment by Kingdorian** [comment]
Great, I think we can merge this now.

**Pull request merged**